app/views/pages/home.html.erb

```diff
@@ -1,2 +1,41 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>ABCheap Landing Page</title>
+</head>
```

Comment:
Don't need any boilerplate (head/DOCTYPE) as this is already given to us in the application.html.erb

Reply:
Need to delete these lines and there closing tags